orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/OortService.groovy

```diff
@@ -43,9 +43,14 @@ interface OortService {
 
   @GET("/applications/{app}/serverGroups/{account}/{region}/{serverGroup}")
   Response getServerGroup(@Path("app") String app,
-                     @Path("account") String account,
-                     @Path("region") String region,
-                     @Path("serverGroup") String serverGroup)
+                          @Path("account") String account,
+                          @Path("region") String region,
+                          @Path("serverGroup") String serverGroup)
+
+  @GET("/serverGroups/{account}/{region}/{serverGroup}")
+  Response getServerGroupByName(@Path("account") String account,
```

Comment:
I think it'd be OK to overload the method name here. Also then you could mark the other `getServerGroup` as `@Deprecated`.